### PR TITLE
Add tooltip support

### PR DESCRIPTION
**Pull request by mikix**
This adds a single tooltip, as a base for further work.

DISCO-746

Screenshot from 2019-04-25 11-15-55

Screenshot from 2019-04-25 11-16-02

### CODE REVIEW

File: src/components/EditCoursePage/EditCourseForm.jsx [hunk 1]
```diff
@@ -117,12 +117,44 @@ export class BaseEditCourseForm extends React.Component {
           <Collapsible
             title={this.formatCourseTitle(title, courseStatuses)}
             key="Test Key"
+            isOpen
```
Comment (mikix) [inline src/components/EditCoursePage/EditCourseForm.jsx]: I just threw this unrelated change in here - it came out of bug bush.

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "eslint": "^5.16.0",
     "font-awesome": "^4.7.0",
     "history": "^4.7.2",
+    "jsx-to-string": "^1.4.0",
```
Comment (mikix) [inline package.json]: This and react-tooltip are used by prospectus already to do this.

File: src/components/FieldHelp/index.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import jsxToString from 'jsx-to-string';
+import ReactTooltip from 'react-tooltip';
+
+
+class FieldHelp extends React.Component {
```
Comment (mikix) [inline src/components/FieldHelp/index.jsx]: This class stolen from prospetus almost verbatem.

**Comment by mikix** [comment]
Oh, tests would actually be appriopriate for this - there is user interaction and all here. I'll add those

File: src/components/EditCoursePage/EditCourseForm.jsx (hunk 1 above)
Comment (Dillon-Dumesnil) [reply src/components/EditCoursePage/EditCourseForm.jsx]: Looks like "Test Key" was originally put in by Jeff and I believe it has to do with it being in a Collapsible and requiring a unique key value. Not sure why it's that and could probably be switched to a uuid or something, but @jlajoie or @bbaker6225  can probably speak better to it.

File: src/components/FieldLabel/index.jsx [hunk 4]
```diff
@@ -10,21 +11,26 @@ const FieldLabel = (props) => {
...
+      {props.helpText && <FieldHelp id={`${props.id}-help`} tip={props.helpText} />}
       {requireText}
     </div>
   );
 };
 
 FieldLabel.defaultProps = {
   className: '',
+  helpText: '',
+  id: 'label', // will not be suitable for production, but useful default for tests
```
Comment (Dillon-Dumesnil) [inline src/components/FieldLabel/index.jsx]: Why not just make it a required field?

Comment (mikix) [reply src/components/FieldLabel/index.jsx]: Because I was being _laaaaazy_. I can fix.

Comment (mikix) [reply src/components/FieldLabel/index.jsx]: Oh actually - because we haven't converted all the field labels to have helpText yet - and the id is only required if you pass helpText.

So maybe I'll just set the default to null and only show helpText if it's set.

File: src/components/EditCoursePage/EditCourseForm.jsx (hunk 1 above)
Comment (jlajoie) [reply src/components/EditCoursePage/EditCourseForm.jsx]: Yeah that shouldn't be this, just make it something unique, sorry for the mistake!

Comment (bbaker6225) [reply src/components/EditCoursePage/EditCourseForm.jsx]: You shouldn't need `key` at all since it's not in an iterator
